htdocs/js/passwordVisibility.js

```diff
@@ -0,0 +1,115 @@
+/* exported PasswordVisibility */
+
+/**
+ * @author Laetitia Fesselier
+ * @version 1.0.0
+ *
+ * Display a password input
+ * with a toggle option to display it in plain text
+ *
+ * For React integration, see React version (jsx/Password.js)
+ */
+
+/**
+ * Collect all input[type='password'] in the DOM
+ * and instantiate PwdInput
+ */
+class PasswordVisibility {
```

Comment:
This seems to be used as (and acts as) a function, there's no reason to make it a class.

Reply:
@driusan  Fixed!